app/models/conference_preference.rb

```diff
@@ -9,13 +8,11 @@ class ConferencePreference < ActiveRecord::Base
 
   private
 
-    def terms_accepted?
+    def terms
       conference = first_conference_id.present? || second_conference_id.present?
       terms_checkbox = terms_of_ticket == '1' && terms_of_travel == '1'
```

Comment:
Technically speaking this is not a "checkbox". How about
```ruby
terms_checked = ...
```
? Expresses more what's inside the variable.